src/main/java/address/sync/cloud/CloudSimulator.java

```diff
@@ -445,7 +445,7 @@ private RemoteResponse getEmptyResponse(int responseCode) {
 
     private List<CloudPerson> filterPersonsByTime(List<CloudPerson> personList, LocalDateTime time) {
         return personList.stream()
-                .filter(person -> !person.getLastUpdatedAt().isBefore(time))
+                .filter(person -> person.getLastUpdatedAt().isAfter(time))
```

Comment:
I don't think this should be changed. This is based on the `since` [parameter](https://developer.github.com/v3/issues/#parameters) of GitHub Issue API where it will return anything **updated at** or after the time.